lib/project_types/script/config/extension_points.yml

```diff
@@ -1,18 +1,18 @@
 discount:
   assemblyscript:
     package: "@shopify/extension-point-as-discount"
-    version: "~0.1.0"
+    version: "~0.2.0"
```

Comment:
Would recommend switching from `~` to `^`. We had to use `~` before because without a minor version number, `^` fully locks whereas with a minor version `^` indicates to not allow the left-most non-zero number to change.

Reply:
Ah, alright. I'll change that